src/libstd/net/tcp.rs

```diff
@@ -72,24 +72,23 @@ pub struct TcpStream(net_imp::TcpStream);
 ///
 /// # Examples
 ///
-/// ```no_run
+/// ```
```

Comment:
NB: I removed the `no_run` because now that the work is being done inside of `process`, which is never called, "running" this code doesn't really do much anyway.